adapters/handlers/rest/configure_api.go

```diff
@@ -926,6 +927,15 @@ func registerModules(appState *state.State) error {
 			Debug("enabled module")
 	}
 
+	if _, ok := enabledModules[modgenerativeanthropic.Name]; ok {
```

Comment:
also please add this module to `defaultGenerative` list (line 704) as I believe this module can be activated by default bc it's a 3rd party module